File: 2013-09-13/FizzBuzz.py
#!/usr/bin/env python

class FizzBuzz():

    def run(self, number=0):
        result = number
        if number % 15 == 0:
            result = "fizzbuzz"
        elif number % 3 == 0:
            result = "fizz"
        elif number % 5 == 0:
            result = "buzz"
        return result

File: 2014-08-19/src/FizzBuzz.js
var FizzBuzz = function(listSize) {
    this.getList = function() {
        var finalList = [];
        for (i=1; i<=listSize; i++) {        
            finalList.push(this.convert(i));
        }
        return finalList;
    }

    this.convert = function(number) {
        var convertedNumber = '';

        if (number % 5 !== 0 && number % 3 !== 0) {
            return number;
        }
       
        if (number % 3 === 0) {
            convertedNumber  = 'fizz';
        }

        if (number % 5 === 0) {
            convertedNumber  += 'buzz';
        }


        return convertedNumber;
    }
};


